/**:
  lights_manager:
    ros__parameters:
      timer_frequency: 10.0
      battery:
        percent:
          window_len: 6
          threshold:
            low: 0.4
            critical: 0.1
        animation_period:
          low: 30.0
          critical: 15.0
        charging_anim_step: 0.1
      plugin_libs:
        - tick_after_timeout_bt_node
      ros_plugin_libs:
        - call_set_led_animation_service_bt_node
        - check_bool_msg_bt_node
        - read_battery_state_bt_node
